plugins/lua/logistics.lua

```diff
@@ -168,8 +168,8 @@ local function print_status()
     print((' - Items designated for trading: %5d'):format(global_sp_stats.total_trade))
     print((' - Items designated for dumping: %5d'):format(global_sp_stats.total_dump))
     print((' - Animals designated for train: %5d'):format(global_sp_stats.total_train))  -- TODO: fix to include all animals designated for training
```

Comment:
this is only concerning animals in cages in stockpiles. I don't think we need to show all animals designated for training here